Sources/CAudioKit/Nodes/Effects/Dynamics/TransientShaperDSP.mm

```diff
@@ -0,0 +1,614 @@
+//
+//  TransientShaperDSP.m
+//  CAudioKit
+//
+//  Created by Evan Murray on 1/6/21.
+//
+
+#include "SoundpipeDSPBase.h"
+#include "ParameterRamper.h"
+#include "soundpipe.h"
```

Comment:
Do we need to include soundpipe.h? It seems like you could just include DSPBase.h and make it inherit from that.